.github/workflows/load-tests.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       changes: ${{ steps.determine-matrix.outputs.changes }}
       # IF you add a new filter it ALSO needs to be here
       services: >-
-        ["account", "graph", "content-publishing", "content-watcher"]
+        ["account-api", "graph-api", "content-publishing-api", "content-watcher"]
```

Comment:
Makes me want to call it `content-watcher-api` for consistency

Reply:
I might consider actually breaking that service up into `-api` and `-watcher` just for consistency, in the future. Gonna leave it for now.